cdc/processor/sinkmanager/manager.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sinkmanager
+
+import (
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/cdc/processor/pipeline"
+)
+
+// Manager manages the sink of a processor.
+// NOTICE: All methods of Manager are thread-safe.
+type Manager interface {
+	// UpdateBarrierTs updates the barrier ts of all tables in the sink manager.
+	// It is used to advance the progress if RedoLog is not enabled.
```

Comment:
Seems barrierTs is used when RedoLog is **enabled**. If RedoLog isn't enabled, only `tableResolvedTs` will be used.

Reply:
I suggest to remove the line. BarrierTs is also used if redo is enabled.